src/garage/envs/multi_env_wrapper.py

```diff
@@ -192,9 +199,9 @@ def step(self, action):
 
         """
         obs, reward, done, info = self.env.step(action)
-        if not self._metaworld_mt:
+        if not self._has_one_hots:
             obs = self._obs_with_one_hot(obs)
-        info['task_id'] = self._active_task_index
+        info['task_id'] = self.active_task_index
```

Comment:
Are these environment infos already present inside metaworld? If so, we probably shouldn't add them.

Reply:
if the task is already in env_infos, why even bother with using this wrapper with MTSAC? Why append a task one-hot here, only to unpack it from obs later in the algorithm?

Reply:
The point of this wrapper, as far as I'm concerned, is to automatically wrap a list of environments so that a naive single-task algorithm can be used as a multi-task algorithm, without any code changes.

Reply:
i agree -- that's why i think there's no reason to use it if there's going to be something called "MTSAC" -- if it's MT, it should just handle multiple envs natively.

Reply:
That's reasonable, but people also want to be able to use single-task algorithms with multiple envs (e.g. MT-PPO and MT-TRPO), which I think is the main reason this wrapper exists. Admittedly, MT-SAC is a weird case of an algorithm half-way adapted to multi-tasking.

Reply:
I still think it's best not to override the existing `task_id`.